spec/ruby/library/objectspace/memsize_of_all_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../../spec_helper'
+require 'objspace'
+
+describe "ObjectSpace.memsize_of_all" do
+  it "returns a non-zero Integer for all objects" do
+    ObjectSpace.memsize_of_all.should be_kind_of(Integer)
+    ObjectSpace.memsize_of_all.zero?.should be_false
```

Comment:
`ObjectSpace.memsize_of_all.should > 0` seems more expressive and clearer, I'll change to that.